src/listener/can_open_listener.py

```diff
@@ -0,0 +1,123 @@
+import canopen
+import logging
+from threading import Thread
+
+from listener.listener import Listener
+from canopen.sdo.exceptions import SdoCommunicationError
+from time import strftime, localtime
+
+
+class CanOpenListener(Listener):
+    ''' This class is being used to listen on a network and mainly
+        tries to push the changes around other nodes to it's interpreter.
+    '''
+    def __init__(self, config, interpreter=None):
```

Comment:
in constructor interface meegeven zodat serial gekozen kan worden

Reply:
ja ik zal het fixen